lerobot/common/policies/act/modeling_act.py

```diff
@@ -200,12 +202,14 @@ def __init__(self, config: ACTConfig):
         self.config = config
         # BERT style VAE encoder with input tokens [cls, robot_state, *action_sequence].
         # The cls token forms parameters of the latent's distribution (like this [*means, *log_variances]).
-        self.use_input_state = "observation.state" in config.input_shapes
+        self.use_robot_state = "observation.state" in config.input_shapes
```

Comment:
FYI: rename because `input_state` was a little too generic, especially now that I've added `use_env_state`